com.unity.multiplayer.mlapi/README.md

```diff
@@ -1,25 +1,21 @@
-[![](https://i.imgur.com/d0amtqs.png)](https://midlevel.github.io/MLAPI/)
```

Comment:
What's the idea of having this mirrored in the package root? I don't see other packages do that. Unity Editor also does not support markdown so this will look pretty ugly when people view this file in our package.

Reply:
The Unity Standards group told us that we should have a readme in the root and one per package, though there's nothing solid on what goes in one readme vs the other. If you want to take a stab at a package-specific readme please do so, but I think having this information in the repo root is preferred as our primary focus is and will always be the sdk package.